packages/bugsnag_flutter/android/src/main/java/com/bugsnag/flutter/BugsnagFlutter.java

```diff
@@ -406,6 +409,26 @@ JSONObject createEvent(@Nullable JSONObject args) throws JSONException {
             event.addMetadata("flutter", (Map<String, Object>) flutterMetadata);
         }
 
+        JSONObject correlation = args.optJSONObject("correlation");
+        if (correlation != null) {
+            try {
+                String traceId = getString(correlation, "traceId");
+                String spanId = getString(correlation, "spanId");
+                if (traceId != null &&
+                        traceId.length() == HEX_LONG_LENGTH * 2 &&
+                        spanId != null &&
+                        spanId.length() == HEX_LONG_LENGTH
+                ) {
+                    long traceIdMostSignificantBits = hexToLong(traceId.substring(0, HEX_LONG_LENGTH));
+                    long traceIdLeastSignificantBits = hexToLong(traceId.substring(HEX_LONG_LENGTH));
+                    long spanIdAsLong = hexToLong(spanId);
+                    event.setTraceCorrelation(new UUID(traceIdMostSignificantBits, traceIdLeastSignificantBits), spanIdAsLong);
+                }
+            } catch(Exception e) {
+                Log.e("BugsnagFlutter", "correlation parsing", e);
```

Comment:
I think that this can be reasonably dropped, logging to the device log just adds noise. This should never happen after-all.

```suggestion
                // ignore the error, the error correlation will be missing
```